js/count-letter-in-string.js

```diff
@@ -0,0 +1,17 @@
+// https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/javascript
+
+function count(string) {
+  const lettersArr = string.split('');
+  let lettersCount = {};
+
+  lettersArr.forEach((_element, i) => {
```

Comment:
Again, do you need the `i` index variable?

Try to see what is the value of:

* `lettersArry[i]`
* `_element`

and if you can use one instead of the other